From 89ea6689111975211d93ea835050abdb9a735e00 Mon Sep 17 00:00:00 2001
From: Yuri Chipchev <yuric@marvell.com>
Date: Wed, 17 Mar 2021 13:11:18 +0200
Subject: [PATCH 1355/1921] arm64: marvell_v8_sdk_defconfig: add modules for
 OVS compilation

Add configuration supported by the kernel 4.14

Change-Id: I4673de2ba355fb129b76fa66a0e205b5823c07e0
Signed-off-by: Yuri Chipchev <yuric@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/48277
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Stefan Chulski <stefanc@marvell.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 arch/arm64/configs/marvell_v8_sdk_defconfig | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/configs/marvell_v8_sdk_defconfig b/arch/arm64/configs/marvell_v8_sdk_defconfig
index 06a502cb54a8..2e86b3250039 100644
--- a/arch/arm64/configs/marvell_v8_sdk_defconfig
+++ b/arch/arm64/configs/marvell_v8_sdk_defconfig
@@ -129,7 +129,10 @@ CONFIG_IP_MULTICAST=y
 CONFIG_IP_PNP=y
 CONFIG_IP_PNP_DHCP=y
 CONFIG_IP_PNP_BOOTP=y
+CONFIG_NET_IPGRE_DEMUX=m
+CONFIG_NET_IPGRE=m
 CONFIG_IPV6=m
+CONFIG_IPV6_GRE=m
 CONFIG_NETFILTER=y
 CONFIG_NF_CONNTRACK=m
 CONFIG_NF_CONNTRACK_EVENTS=y
@@ -244,6 +247,7 @@ CONFIG_DM_ZERO=m
 CONFIG_NETDEVICES=y
 CONFIG_MACVLAN=m
 CONFIG_MACVTAP=m
+CONFIG_GENEVE=m
 CONFIG_TUN=y
 CONFIG_VETH=m
 CONFIG_VIRTIO_NET=y
-- 
2.31.1

